pyro/distributions/transforms/power.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+from torch.distributions import Distribution, constraints
+from torch.distributions.transforms import Transform
+
+
+class PositivePowerTransform(Transform):
+    r"""
+    Transform via the mapping :math:`y = x^{\text{exponent}}`.
```

Comment:
shouldn't this equation look more like line 47? under canonical interpretation this mapping can yield complex numbers, right?

Reply:
oh, like `y = sign(x) * abs(x) ** exponent`?  Sure I'll update the docs.